tests/functional/object/mcg/test_cors_config_elements.py

```diff
@@ -246,39 +348,351 @@ def test_basic_cors_operations(
+        cors_config["CORSRules"][0]["AllowedMethods"] = []
+        for val in self.allowed_methods_list:
+            cors_config["CORSRules"][0]["AllowedMethods"].append(val)
+        cors_config["CORSRules"][0]["AllowedOrigins"] = ["*"]
+        self.apply_cors_on_bucket(
+            cors_config, awscli_pod_session, bucket_name, mcg_obj_session
+        )
+
+        # 8: Perform any request mentioned in allowed HTTP method from any supported origin mentioned on CORS
+        for i in range(3):
+            method = choice(self.allowed_methods_list)
+            origin = self.generate_random_domain_address()[0]
+            sample = TimeoutSampler(
+                timeout=120,
+                sleep=10,
+                func=self.exec_curl_command,
+                endpoint=mcg_obj_session.s3_endpoint,
+                bucket_name=bucket_name,
+                origin=origin,
+                method=method,
+            )
+            sample.wait_for_func_value(self.POSITIVE_RESPONSE)
+
+    @tier2
+    def test_MaxAgeSeconds_and_AllowCredentials_element(
+        self, mcg_obj_session, awscli_pod_session, bucket_factory
+    ):
+        """
+        Test MaxAgeSeconds and AllowCredentials element from CORS operation on bucket
+            step #1: Create bucket and apply basic CORS config with MaxAgeSeconds element in it
+            step #2: Modify MaxAgeSeconds parameter by adding 30 secs in it
+            step #3: Apply basic CORS config on bucket by adding "AllowCredentials" parameter to "False" value
+        """
+        # 1: Create bucket and apply basic CORS config with MaxAgeSeconds element in it
+        bucket_name = bucket_factory(interface="OC")[0].name
+        cors_config = self.create_custom_cors_config(no_of_config=1, method_num=1)
+        self.apply_cors_on_bucket(
+            cors_config, awscli_pod_session, bucket_name, mcg_obj_session
+        )
+        # 2: Modify MaxAgeSeconds parameter by adding 30 secs in it and validate the same
+        new_sec = cors_config["CORSRules"][0].get("MaxAgeSeconds") + 30
+        cors_config["CORSRules"][0]["MaxAgeSeconds"] = new_sec
+        self.apply_cors_on_bucket(
+            cors_config, awscli_pod_session, bucket_name, mcg_obj_session
+        )
+        get_bucket_cors_op = self.exec_cors_command(
+            operation="get-bucket-cors",
+            bucket_name=bucket_name,
+            awscli_pod_session=awscli_pod_session,
+            mcg_obj_session=mcg_obj_session,
+            response=True,
+        )
+        logger.info(get_bucket_cors_op)
+        # Work on below code
+        get_bucket_cors_dict = json.loads(get_bucket_cors_op)
+        assert (
+            cors_config == get_bucket_cors_dict
+        ), "There is mismatch in uploaded CORS config and received CORS config"
+
+        # 3 : Apply basic CORS config on bucket by adding "AllowCredentials" parameter to "False" value
```

Comment:
Please align the " 3 : " part for consistency 
```suggestion
        # 3: Apply basic CORS config on bucket by adding "AllowCredentials" parameter to "False" value
```

Reply:
Fixed